Add Skeleton loading animation to the Wallet selector

johnny9 opened the pull request:
This adds the first skeleton loading section to the desktop wallet. Specifically the WalletBadge at the top left corner that is used for selecting a wallet.


https://github.com/user-attachments/assets/968c8da2-61af-459b-ad62-6b05c1ed7bb8

johnny9 commented:
https://github.com/user-attachments/assets/02294e33-c84b-428a-b420-6f86fc2234aa

Here is an attempt at the gradient shimmer

johnny9 commented:
Update from [1f65d17](https://github.com/bitcoin-core/gui-qml/commit/1f65d17ec8d530eb2e0e08c2e5fc96a0eebcfba2) to [9a53889](https://github.com/bitcoin-core/gui-qml/commit/9a53889d63df5eb94119b279d66ed3dabfd59141)
 - Replaced the Animation in Skeleton with the gradient "shimmer" animation.

GBKS commented:
Looks great (from the video). Just built the code and tried to check it out in the app, but things load so quickly for me that I don't see it. Do I need to load it up with lots of transactions, or does it only show during IBD or so?

Small thing I noticed is that the content of the wallet display wiggles a bit horizontally when the name is shorter/longer than the balance. Could we please align things to the left?

![40185](https://github.com/user-attachments/assets/4d70348c-612d-488d-b1d1-fe10092b8716)

johnny9 commented:
Update from [9a53889](https://github.com/bitcoin-core/gui-qml/commit/9a53889d63df5eb94119b279d66ed3dabfd59141) to [99ca9fd](https://github.com/bitcoin-core/gui-qml/commit/99ca9fde8032177fb54c662cb929bc2ade790d94)
 - Removed semicolons in the qml
 - Removed useless anchor in Item

johnny9 commented:
> Looks great (from the video). Just built the code and tried to check it out in the app, but things load so quickly for me that I don't see it. Do I need to load it up with lots of transactions, or does it only show during IBD or so?
> 
> Small thing I noticed is that the content of the wallet display wiggles a bit horizontally when the name is shorter/longer than the balance. Could we please align things to the left?
> 
> ![40185](https://private-user-images.githubusercontent.com/695901/445942375-4d70348c-612d-488d-b1d1-fe10092b8716.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDc5MzcxMzEsIm5iZiI6MTc0NzkzNjgzMSwicGF0aCI6Ii82OTU5MDEvNDQ1OTQyMzc1LTRkNzAzNDhjLTYxMmQtNDg4ZC1iMWQxLWZlMTAwOTJiODcxNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNTIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDUyMlQxODAwMzFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wMzgwNDM4MmFmZmJhNWRlY2Y5MDUzZmQ2YjU1NzczNmNjNWQxYTU3MWE0NzFiNzE3MWQ1OGZhNDczYjQ1OThiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9._rgutbpPfPimDu-kdQFBRluUdrC-h5Vjc47M46CXZM0)

Created an issue to track this. It should be possible to fix the alignment.
